packages/studioCMS/src/integrations/studioCMSDashboard/routes/databaseSetup/main.astro

```diff
@@ -1,36 +1,34 @@
-      <div class="py-2">
-          <text class="text-lg py-2">
-            <text class="font-bold">What is StudioCMS?</text> 
-              <br />
-              StudioCMS is a simple, easy to use, and fast CMS for Astro. It's built with the latest technologies and is designed to be as simple as possible. It's also open source and free to use. 
-          </text> 
-          <div class="my-2 divider" />
-          <text class="text-lg py-2">
-            <text class="font-bold">How do I use it?</text> 
-              <br />
-              To use StudioCMS, you need to have a basic understanding of Astro and how it works. As well as StudioCMS. You can use the CMS to create, edit, and delete posts. You can also use it to manage your site's settings and more.
-          </text> 
-          <div class="my-2 divider" />
-          <text class="text-lg py-2">
-            <text class="font-bold">What's next?</text> 
-              <br />
-              The next step is to start by filling in your site details below!
-          </text> 
-          <div class="my-2 divider" />
-          <text class="text-lg py-2">
-            <text class="font-bold">Problems? or Need Help?</text> 
-              <br />
-              If you have any problems or need help, you can visit the <a href="https://github.com/astrolicious/studiocms" target="_blank" class="text-blue-500">GitHub Repository</a> or join the <a href="https://chat.astrolicious.dev" target="_blank" class="text-blue-500">Discord Server</a> for help.
-          </text> 
+      <div class="flex flex-col py-2 w-full gap-4">
+        <Card>
+          <text class="font-bold">What is StudioCMS?</text> 
+          <Divider class="divider" />
+          StudioCMS is a simple, easy to use, and fast CMS for Astro. It's built with the latest technologies and is designed to be as simple as possible. It's also open source and free to use. 
+        </Card>
+        <Card>
+          <text class="font-bold">How do I use it?</text> 
+          <Divider class="divider" />
+          To use StudioCMS, you need to have a basic understanding of Astro and how it works. As well as StudioCMS. You can use the CMS to create, edit, and delete posts. You can also use it to manage your site's settings and more.
+        </Card>
+        <Card>
+          <text class="font-bold">What's next?</text> 
+          <Divider class="divider" />
+          The next step is to start by filling in your site details below!
+        </Card>
+        <Card>
+          <text class="font-bold">Problems? or Need Help?</text> 
+          <Divider class="divider" />
+          If you have any problems or need help, you can visit the <a href="https://github.com/astrolicious/studiocms" target="_blank" class="text-blue-500">GitHub Repository</a> or join the <a href="https://chat.astrolicious.dev" target="_blank" class="text-blue-500">Discord Server</a> for help.
+        </Card>
       </div>
       
       <br />
```

Comment:
br!

Reply:
This was not a CSS/Layout update, this was a Shoelace.style update